### ISSUE TITLE
Allow to order the http method in the spec

### ISSUE THREAD

**Issue by dcu** [Apicurio/apicurio-studio#957]
In our case we want to display the endpoints to create the resource first and by default apicurio is putting them at the end.

**Comment by EricWittmann** [comment]
Thanks for the feedback.  You actually mean ordering of the HTTP methods for a given path?

**Comment by dcu** [comment]
yes, that's it. So we can give an order to the generated documentation
